package.json

```diff
@@ -141,6 +141,9 @@
   "xo": {
     "ignore": [
       "cli.js"
-    ]
+    ],
+		"rules": {
+			"no-loop-func": 1
```

Comment:
Had to insert this rule, because it does not like https://github.com/sindresorhus/ava/blob/cleanup-sequence/lib/sequence.js#L42:

```js
for (...) {
  return fn().then(function () {
    // this anonymous function is a problem for linter
  });
}
```